docs: add English versions of Cómo-usar-git-en-consola.md and Cómo-actualizar-y-configurar-tu-copia-local-de-omegaup-antes-de-hacer-cambios.md

thunderbird7413 opened the pull request:
This PR adds the English versions of two documentation pages:

- Cómo-usar-git-en-consola.md → How-to-use-git-in-console.md
- Cómo-actualizar-y-configurar-tu-copia-local-de-omegaup-antes-de-hacer-cambios.md → How-to-update-and-configure-your-local-copy.md

Both files follow the structure and formatting of the original Spanish documents and the URLs are pointed correctly wherever required.

This improves documentation accessibility for non-Spanish-speaking contributors.

Fixes: #8585

pabo99 commented on frontend/www/docs/Cómo-usar-git-en-consola.md:
```diff
@@ -28,4 +28,4 @@ Es muy probable que ya lo tengas instalado, para revisar abre la consola (Termin
 
-¡Estás listo! Ahora sigue los pasos de este artículo https://github.com/omegaup/omegaup/blob/main/frontend/www/docs/C%C3%B3mo-actualizar-tu-copia-local-de-omegaup-antes-de-hacer-cambios.md
\ No newline at end of file
+¡Estás listo! Ahora sigue los pasos de este artículo https://github.com/omegaup/omegaup/blob/main/frontend/www/docs/C%C3%B3mo-actualizar-y-configurar-tu-copia-local-de-omegaup-antes-de-hacer-cambios.md
```
This file can be removed as well

pabo99 reviewed (commented):
Sorry for the delayed response. I’ve left you some suggestions to help get this PR ready

thunderbird7413 commented:
I have incorporated all the requested changes and updated the PR.
Kindly review the latest changes and let me know if any further adjustments are needed.

pabo99 reviewed (approved):
Thanks, @thunderbird7413!
